### ISSUE TITLE
S001: Students can create their Internship Information before applying

### ISSUE THREAD

**Issue by bkleinen** [imimaps/imimaps#85]
... such that Admin can comment on it during the registration process and this information can be seen later on

- enter basically all information that goes on the registration form
http://imi-bachelor.htw-berlin.de/files/Stg/IMI/Registration-Internship-IMI-B-en.pdf

**Comment by ViviZa** [comment]
login process should be already done before starting this story !
-(re)design form , button "create internship" , search for (if company already exist in database) company or create company
